apiary.apib

```diff
@@ -2618,6 +2618,124 @@ Your Accredible account is represented as an Issuer within the API.
             }
         }
 
+# Group Data Exports
+
+## Data Exports [/v1/data_exports]
+
+### View All Data Exports [GET]
+
+Returns list of data exports of departments accessible via the used API_KEY.
+
++ Request (application/json)
+
+    + Headers
+            Authorization: Token token=YOUR API_KEY
+
++ Response 200 (application/json)
+
+        {
+           "data_exports": [
+                    {
+                        "id": 1,
```

Comment:
like the earlier conversation on `#project-data-export` - would this not be better as a UUID?

Reply:
Given our other endpoints all expose internal IDs, this feels like a v2 issue to me.